README.md

```diff
@@ -1,13 +1,23 @@
 # Kabanero Foundation
 
-The Kabanero Foundation Instance is a deployment of a Kabanero collection in a specific Kubernetes cluster.  The Kabanero operator is leveraged to install and operate the various curated infrastructure and artifacts as defined by the Kabanero collection.
+The Kabanero Foundation Instance is a deployment of a set of Kabanero collections in a specific Kubernetes cluster.  The Kabanero operator is leveraged to install and operate the various curated infrastructure and artifacts as defined by each of the Kabanero collections. 
```

Comment:
I'd prefer talk about Kabanero Instances vs "Foundation" instances.  We introduced the notion of a Foundation before to account for all the 'infrastructure' components that get installed like tekton, istio, etc.  I feel we should just short and call this a  "Kabanero Instance".

Also we use the word Kabanero a lot.  In the last sentence just abreviate to "collections" vs "Kabanero collections".  You already indicate the full name above.